python/folderify/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        lgarron folderify 1.2.3 v
+github.tarball_from archive
+
+categories          python amusements
+platforms           darwin
+license             MIT
+
+maintainers         @harens \
+                    @lgarron
```

Comment:
Does @lgarron agree to be co-maintainer on this port?

Reply:
Also can you add emails and/or `openmaintainer`?

Reply:
Thanks for reviewing this PR. I haven't received a response from @lgarron in this issue https://github.com/lgarron/folderify/issues/43, so I presume he doesn't want to be a co-maintainer.